### ISSUE TITLE
Mode Value in JSON Object

### ISSUE THREAD

**Issue by hannsadrian** [derhuerst/live-icomera-position#1]
In your readme "mode" is commented as unknown. My speculation is that the icomera system internally uses "gpsd" a widely adopted gps module driver. They use mode as a value for accuracy of the gps position.
Here are mode values as I know them from the docs:
0 = no value seen.
1 = no position.
2 = 2D position, no altitude.
3 = 3D position, altitude present.

Read more in GPSD specification: https://gpsd.gitlab.io/gpsd/gpsd_json.html

**Comment by derhuerst** [comment]
> In your readme "mode" is commented as unknown. My speculation is that the icomera system internally uses "gpsd" a widely adopted gps module driver. They use mode as a value for accuracy of the gps position.

Thanks for documrnting this here, sounds plausible!

> Here are mode values as I know them from the docs:
> 0 = no value seen.
> 1 = no position.
> 2 = 2D position, no altitude.
> 3 = 3D position, altitude present.

These are (at least) the values mentioned in the gpsd docs. You didn‘t get them from Icomera docs, right?

**Comment by hannsadrian** [comment]
Yes, I use gpsd personally and as I glanced at the readme the mode value seemed familiar.
I have absolutely no experience with Icomera at all but in my projects it was always very convenient and easy to use gpsd.
Are there any Icomera docs publicly available that give a hint whether they're using gpsd?

**Comment by derhuerst** [comment]
> Are there any Icomera docs publicly available that give a hint whether they're using gpsd?

I have no idea. 🤷

**Comment by derhuerst** [comment]
I would happily merge a PR that adds this to the documentation, with the note that it is not confirmed.
